particle-iot/particle-cli
# feature/sc-107205/remove-build-step

## Description

Removes build step, includes and targets uncompiled source in published artifacts


## How to Test

1. Pull down this branch (`git pull && git checkout chore/dependencies`)
2. Reinstall dependencies (`npm run reinstall`)
3. Remove now unused compiled source files: `rm -rf ./dist`
4. Run commands as normal: `npm start -- help` ([docs](https://github.com/particle-iot/particle-cli#running)) 
5. Observe [CI passing](https://app.circleci.com/pipelines/github/particle-iot/particle-cli/98/workflows/926d90a6-a5b5-4cdc-9e74-143adf8db9c1)

**Outcome**

CLI should work without error targeting the compiled files


## Related Issues / Discussions

https://app.shortcut.com/particle/story/107205
https://app.shortcut.com/particle/story/107005


## Completeness
